changelog_unreleased/javascript/14862.md

```diff
@@ -0,0 +1,11 @@
+#### Support experimental `await using` syntax (#14861, #14862 by @sosukesuzuki)
+
+Support `await using` for [Explicit Resource Management](https://github.com/tc39/proposal-explicit-resource-management/) proposal.
+
+<!-- prettier-ignore -->
+```jsx
+{
+  await using db = await Database.connect("//my-db.sql");
+  db.query("INSERT INTO tools (name, version) VALUES ('Prettier', '3.0.0')");
+} // At the end of the block, the db connection will be closed asynchronously
+```
```

Comment:
Let's merge this into #13752? since it haven't been released.

Reply:
[9068cb7](https://github.com/prettier/prettier/pull/14862/commits/9068cb7ba31ac4ebb13c947ec21534c4c618eb0e)